fbv is writing to a 32 bit buffer setting the alpha bit
per each pixel to be zero instead of real value from
the picture.

--- fbv-1.0b/fb_display.c	2016-03-09 17:00:07.499180395 -0600
+++ fbv-1.0b_mod/fb_display.c	2016-03-09 17:01:19.719328395 -0600
@@ -375,7 +375,8 @@
 	    *cpp = 4;
 	    i_fbbuff = (unsigned int *) malloc(count * sizeof(unsigned int));
 	    for(i = 0; i < count ; i++)
-		i_fbbuff[i] = ((rgbbuff[i*3] << 16) & 0xFF0000) |
+		i_fbbuff[i] = 0xFF000000 | 
+			    ((rgbbuff[i*3] << 16) & 0xFF0000) |
 			    ((rgbbuff[i*3+1] << 8) & 0xFF00) |
 			    (rgbbuff[i*3+2] & 0xFF);
 	    fbbuff = (void *) i_fbbuff;
